src/test/policy_fee_tests.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <amount.h>
+#include <policy/fees.h>
+
+#include <test/util/setup_common.h>
+
+#include <boost/test/unit_test.hpp>
+
+BOOST_FIXTURE_TEST_SUITE(policy_fee_tests, BasicTestingSetup)
+
+BOOST_AUTO_TEST_CASE(FeeRounder)
+{
+    FeeFilterRounder fee_rounder{CFeeRate{1000}};
+
+    // check that 1000 rounds to 974 or 1071
+    std::set<CAmount> results;
+    while (results.size() < 2) {
+        results.emplace(fee_rounder.round(1000));
+    }
+    BOOST_CHECK_EQUAL(*results.begin(), 974);
```

Comment:
What is the purpose of fee rounder and why it will return 974 and 1071?

Reply:
There are privacy concerns with deanonymizing a node by the fact that it is broadcasting identifying information about its mempool min fee. To help ameliorate this concern, the implementation quantizes the filter value broadcast with a small amount of randomness, in addition, the messages are broadcast to different peers at individually randomly distributed times.

Source: https://github.com/bitcoin/bips/blob/master/bip-0133.mediawiki#considerations